lock/src/test/java/org/infinispan/lock/ClusteredLockSplitBrainTest.java

```diff
@@ -30,16 +31,13 @@
 
    public ClusteredLockSplitBrainTest() {
       this.numMembersInCluster = 6;
-      //this.cacheMode = CacheMode.REPL_SYNC;
+      this.cacheMode = CacheMode.REPL_SYNC;
    }
 
    @Override
    protected void createCacheManagers() throws Throwable {
       ConfigurationBuilder dcc = cacheConfiguration();
-      dcc.clustering().cacheMode(cacheMode).partitionHandling().whenSplit(partitionHandling).mergePolicy(mergePolicy);
-      if (biasAcquisition != null) {
-         dcc.clustering().biasAcquisition(biasAcquisition);
-      }
+      dcc.clustering().cacheMode(cacheMode).partitionHandling().whenSplit(partitionHandling);
```

Comment:
I'd remove the partition handling stuff as well, and hard-code the cache mode. That way you can set `this.cacheMode = null` in the constructor, and `REPL_SYNC` won't appear in the test reports (because the default cache isn't used at all).

Reply:
when I remove partition handling, test fails

Reply:
ok for the rest